app/controllers/administrator_dashboards_controller.rb

```diff
@@ -2,7 +2,7 @@ class AdministratorDashboardsController < ApplicationController
   before_action :authenticate_user!
 
   def show
-    @active_campaigns = Campaign.active.order(name: :asc)
+    @active_campaigns = Campaign.active.order(name: :asc).limit(9)
```

Comment:
Is this a stub before adding pagination?

Reply:
No.. it's just a limit to the admin dashboard. We'll enhance it later